data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -82,6 +84,7 @@ def transform(self) -> None:
         logger.info("Transforming National Risk Index Data")
 
         NRI_TRACT_COL = "TRACTFIPS"  # Census Tract Column in NRI data
+        TRACT_COL = self.GEOID_TRACT_FIELD_NAME  # Census Tract column name
```

Comment:
Why are you going back to using TRACT_COL? Are you sure you branched off of `mega-tracts` / the latest on `main`?